src/plugins/intel_cpu/src/plugin.h

```diff
@@ -46,15 +46,19 @@ class Engine : public InferenceEngine::IInferencePlugin {
 
     void ApplyPerformanceHints(std::map<std::string, std::string> &config, const std::shared_ptr<ngraph::Function>& ngraphFunc) const;
 
+    int GetModelPreferThreads(const std::shared_ptr<ngraph::Function>& ngraphFunc) const;
+
     struct StreamCfg {
         int num_streams;
         int big_core_streams;          // Number of streams in Performance-core(big core)
+        int big_core_logic_streams;    // Number of streams in Performance logical core(big core)
         int small_core_streams;        // Number of streams in Efficient-core(small core)
         int threads_per_stream_big;    // Threads per stream in big cores
         int threads_per_stream_small;  // Threads per stream in small cores
         int small_core_offset;
     };
     enum StreamMode { DEFAULT, AGGRESSIVE, LESSAGGRESSIVE };
```

Comment:
Please use one definition of `enum StreamMode { DEFAULT, AGGRESSIVE, LESSAGGRESSIVE };` in both CPU plugin and StreamsExecutor

Reply:
Ok, I remove this definition from CPU plugin.